### DISCUSSION TITLE
Restore menus from JSON file

### DISCUSSION THREAD

**Discussion by VanilleAngeles** [Schneegans/Fly-Pie#188] [Fly-Pie Q&A] [answered]
Hello everyone,
I use 2 distros on a same PC, Manjaro with Gnome 41.1 and Debian 11 with Gnome 3.38.
I have backuped Fly-Pie configuration on a network device (from Debian).
When I want to duplicate my configuration from Debian to Manjaro (Restore menus fonction), the answer is 
`Could not import the menu configuration. The JSON file should contein an array of menu configurations`.
And if I want to restore the JSON file on the original distro (Debian), same message.
Where is my mistake ?
Thanks.

**Comment by Schneegans** [comment, marked as answer]
Just tested it on my end and it seems to work. Maybe you mixed up the configuration files? There are two files you can export / import. On the one hand there are the appearance settings (exported and imported from the "Settings" page of the preferences dialog), and on the other hand there is the menu configuration (exported and imported from the "Menu Editor" page of the preferences dialog).

The appearance settings contain a JSON object (the first character in the file should be a `{`) while the menu configuration contains an array (the first character in the file should be a `[`).

**Reply by VanilleAngeles** [reply]
I have only one response: PERFECT. Thanks a lot and long life to Fly-Pie.
👍
